### PR TITLE
added blob and clob support for data insertion

### PR DESCRIPTION
**Pull request by ashutoshrnjn**
#### Related issues

- connect to <link_to_referenced_issue>

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Comment by ashutoshrnjn** [comment]
Hi @virkt25 @jannyHou @b-admike @dhmlau, Can you all please take some time to review this PR.
Thanks

**Comment by dhmlau** [comment]
@slnode test please

File: lib/ibmdb.js [hunk 1]
```diff
@@ -314,14 +314,31 @@ function dateToIBMDB(val) {
...
+  const transformedValue = this.transformColumnValue(prop, val);
+  if (val == null || !prop || !prop.db2) {
+    return transformedValue;
+  }
+  // db2.datatype needs to be defined in User Defined Model Definition
+  switch (prop.db2.dataType) {
+    case 'BLOB':
+      return {CType: 'BLOB', DataType: 'BLOB', Data: transformedValue};
+    case 'CLOB':
+      return {DataType: 'CLOB', Data: transformedValue};
```
Comment (virkt25) [inline lib/ibmdb.js]: Should there be a `CType` here as well?

Comment (ashutoshrnjn) [reply lib/ibmdb.js]: @virkt25 @dhmlau @sam-github @rmg : please let me know what is needed to merge this PR.

Comment (sam-github) [reply lib/ibmdb.js]: /cc @bajtos

**Pull request merged**